src/samv71_serial_ccsds/samv71_serial_ccsds.c

```diff
@@ -228,7 +230,8 @@ SamV71SerialCcsdsInit_rx_handler(samv71_serial_ccsds_private_data *const self) {
   self->m_uart_rx_handler.lengthArg = self;
   self->m_uart_rx_handler.characterArg = self;
   self->m_uart_rx_handler.targetCharacter = STOP_BYTE;
-  self->m_uart_rx_handler.targetLength = Serial_CCSDS_SAMV71_RECV_BUFFER_SIZE;
+  self->m_uart_rx_handler.targetLength =
+      Serial_CCSDS_SAMV71_RECV_BUFFER_SIZE / 2;
```

Comment:
Why?

Reply:
To minimize the chance of FIFO overflow. If an interrupt is triggered when FIFO is full, there is a high probability that FIFO will be overflowed. It requires only that one additional byte arrives before pulling data from the FIFO. To be honest, in this situation, the overflow happens every time. It is better to have a "safety buffer", in case many bytes arrive while polling data.
I have tested it a couple of times. Not having the "safety buffer" always caused the overflow error. After adding this buffer space, the overflow error never occurred in tests.